### PR TITLE
feat: Allow entering agenda text directly

### LINKED ISSUE #6532: Allow entering text directly instead of uploading a file for the agenda
### Description

It would save a step to have a text field for editing an agenda inline instead of having to first create a file on my computer and upload it.

image

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by pselkirk**
Fixes #6532

### CODE REVIEW

**Comment by pselkirk** [comment]
For whatever reason (because I don't know javascript), the script that shows the appropriate file/text input box (taken from complete-review.js) doesn't hide the label of the hidden field.

**Comment by jennifer-richards** [comment]
> For whatever reason (because I don't know javascript), the script that shows the appropriate file/text input box (taken from complete-review.js) doesn't hide the label of the hidden field.

The issue is that the `bootstrap_form` call is rendering the fields like (paraphrasing)
```html
<div class="mb-3">
  <label class="form-label" ...>Agenda text</label>
  <textarea id="id_content" name="content" ...></textarea>
</div>
```
and you are selecting for `[name="content"]`, which selects only the `<textarea/>`.

Easiest thing to do is to change line 54 of the template to `var row = form.find(selector).parent();` so you hide the whole `<div>...</div>` wrapper.  It's somewhat fragile, but I don't see a way to more directly label and select the wrapper `<div/>`.

Also, not sure if you're ready for review, but the JS code should go in its own module like is done for `complete-review.js` so that it can be packaged properly.

**Comment by pselkirk** [comment]
> `var row = form.find(selector).parent();`

Thanks, that was the nudge I needed.

> the JS code should go in its own module like is done for `complete-review.js` so that it can be packaged properly.

Yeah, I just wanted to get the JS working before going through the packaging step.

**Review by jennifer-richards — CHANGES_REQUESTED** [review]
Suggestion inline - I'm open to discussion if you think it's preferable to stick with the status code checks as they are

File: ietf/meeting/tests_views.py [hunk 1]
```diff
@@ -6223,6 +6223,45 @@ def test_upload_minutes_agenda_interim(self):
...
+        login_testing_unauthorized(self,"secretary",url)
+        r = self.client.get(url)
+        self.assertEqual(r.status_code, 200)
+        q = PyQuery(r.content)
+        self.assertIn('Upload', str(q("Title")))
+        self.assertFalse(session.sessionpresentation_set.exists())
+
+        test_text = 'Enter agenda from scratch'
+        r = self.client.post(url,dict(submission_method="enter",content=test_text))
+        self.assertEqual(r.status_code, 302)
```
[the remark this answers is not in the document]
Comment (pselkirk) [reply ietf/meeting/tests_views.py:6238]: Sure, it's an easy enough change, and it's a more complete test.

**Review by jennifer-richards — APPROVED** [review]

**Pull request merged**